modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/base/ComposableBase.kt

```diff
@@ -164,7 +164,9 @@ fun ComposableTaskerInputFieldList(
     fieldContents: List<TaskerInputFieldState.Content<*>>,
     onFinish: () -> Unit
 ) {
-    Box(modifier = Modifier.fillMaxHeight()) {
+    Box(modifier = Modifier
+            .padding(10.dp)
```

Comment:
nitpick, but material guidelines suggest using an 8dp grid https://m1.material.io/layout/metrics-keylines.html#metrics-keylines-baseline-grids

Reply:
TIL thanks for sharing.